malsim/wrappers/gym_wrapper.py

```diff
@@ -23,13 +26,28 @@ def __init__(self, scenario_file: str, **kwargs) -> None:
         self.render_mode = kwargs.pop('render_mode', None)
 
         # Create a simulator from the scenario given
-        self.sim, _ = create_simulator_from_scenario(scenario_file, **kwargs)
+        attack_graph, agents = load_scenario(scenario_file, **kwargs)
+        self.sim = MalSimVectorizedObsEnv(MalSimulator(attack_graph))
```

Comment:
I find it confusing that self.sim actually holds an environment, or at least that's what its name says, `MalSimVectorizedObsEnv`. If this is truly an environment, then `self.sim` should be renamed. Could it be that we actually want custom gym envs to subclass the malsim vectorized env as well? In any case, it is a bit messy right now I think. From a gym env, you could do `self.sim.sim.<MalSimulator_thing>` which is confusing.